### DISCUSSION TITLE
Airlow connection to Desktop applications

### DISCUSSION THREAD

**Discussion by Bhavanakamidi** [apache/airflow#28877] [General]
Hi Team,how can we connect unix applications and Desktop applications from Airflow?
Can we connect xterm from airflow??
Can we connect xwin32 from airflow??
My requirement is connect to xwin32 from Airflow and launch unix desktop and connect to xterm and trigger script....how can we do from Airflow is it possible?? Or is there any alternative way?

**Comment by o-nikolas** [comment]
Do you really need to start a desktop session to run the script? Perhaps using the [SSH Operator](https://airflow.apache.org/docs/apache-airflow-providers-ssh/stable/_api/airflow/providers/ssh/operators/ssh/index.html) is good enough to execute the script on the remote host?
